.travis.yml

```diff
@@ -43,6 +43,6 @@ jobs:
       name: report_coverage
       # Generating test coverage report and publishing to Codacy
       script:
-        - mvn jacoco:prepare-agent test jacoco:report -B
-        - curl -Ls -o codacy-coverage-reporter-assembly.jar https://github.com/codacy/codacy-coverage-reporter/releases/download/6.0.2/codacy-coverage-reporter-6.0.2-assembly.jar
-        - java -jar codacy-coverage-reporter-assembly.jar report -l Java -r target/site/jacoco/jacoco.xml
+        - mvn cobertura:cobertura -B
+        - curl -LSs $(curl -LSs https://api.github.com/repos/codacy/codacy-coverage-reporter/releases/latest | jq -r '.assets | map({content_type, browser_download_url} | select(.content_type | contains("application/octet-stream"))) | .[0].browser_download_url') -o codacy-coverage-reporter-assembly.jar
```

Comment:
Is this change really required? Why was it modified in the master. 
@e-pettersson-ericsson / @a-breid-ericsson  can you comment if this change is required in merging the maintenance branch to master?

Reply:
travis.yml 14 months ago:
https://github.com/eiffel-community/eiffel-intelligence/blame/2.0.0-maintenance/.travis.yml

travis.yml 12 months ago, on master: 
https://github.com/eiffel-community/eiffel-intelligence/blame/master/.travis.yml

I guess taking the latest changes would be the safest, but Codacy does not seem to be enabled at all anymore, so a separate issue should be created and investigated how to enable it again.